docs/.vitepress/config.js

```diff
@@ -165,6 +165,7 @@ const config = {
           text: "Customize Avo",
           items: [
             {text: "Customization options", link: "/3.0/customization"},
+            {text: "Eject", link: "/3.0/eject"},
```

Comment:
```suggestion
            {text: "Eject views", link: "/3.0/eject-views"},
```

And please lets move above `Routing`